web/impact/impact/tests/test_user_list_view.py

```diff
@@ -507,6 +507,21 @@ def test_updated_at_after_datetime_filter(self):
             assert _contains_user(updated_exactly, response.data)
             assert _contains_user(updated_after, response.data)
 
+    def test_updated_at_filter_when_same_day(self):
```

Comment:
drive-by comment: there is a lot of duplication in these tests. While we're here, it would be great if we could clean that up and make it pretty.